openbb_terminal/helper_funcs.py

```diff
@@ -1953,6 +1972,100 @@ def str_date_to_timestamp(date: str) -> int:
     return date_ts
 
 
+def update_news_from_tweet_to_be_displayed() -> str:
+    """Update news from tweet to be displayed.
+
+    Returns
+    -------
+    str
+        The news from tweet to be displayed
+    """
+    global LAST_TWEET_NEWS_UPDATE_CHECK_TIME
+
+    # Check whether it has passed a certain amount of time since the last news update
+    if (
+        datetime.now(pytz.utc) - LAST_TWEET_NEWS_UPDATE_CHECK_TIME
+    ).seconds > obbff.TOOLBAR_TWEET_NEWS_SECONDS_BETWEEN_UPDATES:
```

Comment:
You probably want to use total_seconds here instead.  Seconds will only give you a number from 0-60 based on the timestamps.